apple_framework_relative_headers.bzl

```diff
@@ -0,0 +1,67 @@
+# Forked from
```

Comment:
Because it's forked does that mean we don't need a copyright?

Reply:
I don't see the same method definition elsewhere and it seems substantial enough that I think it warrants a header. Since both this repo and material-internationalization-ios are under the material-foundation organization, the copyright header would be the same in any case.  I'm going to add it for now (to move forward) and follow-up with @ianegordon to confirm that it makes sense to him as well.